src/main/kotlin/pt/up/fe/ni/website/backend/model/Post.kt

```diff
@@ -0,0 +1,16 @@
+package pt.up.fe.ni.website.backend.model
+
+import java.util.Date
+import javax.persistence.Entity
+import javax.persistence.GeneratedValue
+import javax.persistence.Id
+
+@Entity
+class Post(
+    var title: String,
+    var body: String,
+    var href: String,
```

Comment:
I meant to remove it. I meant the `href` as an example of just creating links with HTML code, which is inside the body string

Reply:
@BrunoRosendo I was under the impression that these were blog posts. If so, wouldn't it make sense to actually have an href in the model. For instance, I could create a blog post at https://ni.fe.up.pt/blog/how-we-made-lives-for-up-students-easier. In this case, the href would be the part after `blog/`. Are you sure it's best to remove it?

Reply:
Oh I was talking about something else. We're using the ID for the GET request. I agree that looks better and we could maybe query it that href instead of the ID. I would say create an issue for it. What do you think?

Reply:
Yes, that sounds good!